TechnologyAdvice/laravel-drafts
# ES-6130 Fix bug causing drafts to not inherit the initial created_at

Please follow our [PR Guidelines](https://technologyadvice.atlassian.net/wiki/spaces/EN/pages/962560174/PR+Documentation+Review+Guidelines) for submitting or reviewing a PR

### Description
When a draft was created, it would have a new created_at rather than using the created_at from the original record. This fixes that bug.

### Related issue(s)
https://technologyadvice.atlassian.net/browse/ES-6130

### Related PR(s)

### How to test / repro
Supposedly you can follow these steps (in products repo) to link this repo locally, but it didn't work for me. Alternatively, just copy the HasDraft's line into vendor/technologyadvice/laravel-drafts/src/Concerns/HasDrafts.php.

1. Run unit tests
2. Create a new product and verify it is successfully created
3. Edit new product and verify the created_at has not changed in the DB (since UI doesn't show timestamps)

### Screenshots

### Changes include

### Checklist
- [ ] If this is a bugfix, I've added testing to check this in the future
- [ ] I have added appropriate logging at the correct levels
